lib/spack/llnl/util/filesystem.py

```diff
@@ -119,10 +119,7 @@ def file_command(*args):
     try:
         file_cmd = which("file", required=True)
     except CommandNotFoundError as e:
-        if is_windows:
-            raise CommandNotFoundError("`file` utility is not available on Windows")
-        else:
-            raise e
+        raise e
```

Comment:
This entire try/catch can be removed, it was only added to implement the additional error reporting on Windows.

Reply:
True. I removed that.